website/src/content/blog/roadmap-2024.md

```diff
@@ -0,0 +1,360 @@
+
+Moving small projects from ESLint/Prettier is easy, but moving **big** code bases is challenging and time-consuming. This is a big point of friction in Biome.
+
+Users have different needs though, so it will be impossible to satisfy all of them. We want to make sure that all features and contributions to our project [embrace our philosophy](/internals/philosophy/) and provide the best experience by default.
+
+### Main area of focus
+
+1. Help users to move to Biome
+2. Expand Biome's language support, so Biome tools can span more of the web ecosystem
+3. Deepen Biome's existing capabilities to offer more functionalities
+4. Plugins
+5. Transformations
+6. Community and content
+
+### Help users to move to Biome
+
+- Offer guides on our website to users who want to migrate from Prettier (CLI commands and configuration)
+- Offer guides on our website to users who want to migrate from ESlint (CLI commands and configuration)
+- Offer a section on our website that shows a mapping of the ESLint rules to our rules
+- Offer commands to ease the transition
+  - A command called `biome migrate prettier` that will read `.prettierrc` and `.prettierignore`, will update the `biome.json` file (or create it) with the configuration coming from the Prettier files.
+  - A command called `biome migrate eslint` that will read the JSON configuration of Eslint and the ignore file. There will be expectations and limitations.
+
+
+### Expand Biome's language support
+
+CSS is our next language of focus and is making good progress. HTML and Markdown will follow. Follow our [up-to-date page](/internals/language-support) to keep up with the progress of our work.
+
+The CSS language will enable a lot of work and experimentation: CSS formatting and linting, and we will port some of the lint rules from `styelelint`. A new area of experimentation is cross-linting.
+
+The idea of cross-linting can be explained with an example: compute the CSS styles/classes defined in a project, and warn a user when said styles aren't used inside JSX/HTML files.
+
+Plus, we unlock another area of experimentation, which is embedded formatting.
+
+HTML and Markdown will be our next languages of focus. HTML will enable us to parse other variants of HTML that are popular in the frontend ecosystem: [Vue](https://vuejs.org/), [Svelte](https://svelte.dev/) and [Astro](https://astro.build/). This would require some exploration on how to represent super languages of HTML.
+
+### Deepen Biome's existing capabilities to offer more functionalities.
+
+- Project analysis and dependency resolution
+- Type system
+- CLI
+
+#### Project analysis and dependency resolution
+
+We will provide lint rules that can read the manifest and do operations over it, such as invalid licenses.
```

Comment:
The end of the sentence don't make sense to me.
Which operations?

What about:

```suggestion
We will provide lint rules that can read the manifest, detect errors such as invalid licenses.
```